ljps-frontend/pages/skill/createskillbutton.js

```diff
@@ -0,0 +1,157 @@
+import axios from "axios";
+import React from "react";
+import { useState, useRef, useEffect } from "react";
+import { validateLength } from "../../util/validation/index";
+
+export default function CreateSkillButton() {
+  const [skillName, setSkillName] = useState("");
+  const [skillDescription, setSkillDescription] = useState("");
+  const [showError, setShowError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const modal = useRef();
+  const toast = useRef();
+
+  function handleSubmit(e) {
```

Comment:
Either put this function outside CreateSkillButton or useCallBack for the function. Or handleSubmit function will get redeclared every time there's a change state.